### ISSUE TITLE
worldedit-bukkit-7.2.11-beta-01 results in java.lang.NumberFormatException from long(?) version number

### ISSUE THREAD

**Issue by amalon** [PlayPro/CoreProtect#245]
Running a beta version of worldedit: worldedit-bukkit-7.2.11-beta-01.jar gives a longish version number string "0162000877589", which results in an exception when parsing it to an integer from Util.loadWorldEdit:

```
[11:42:18 INFO]: [WorldEdit] Enabling WorldEdit v7.2.11-beta-01+6200-0877589
...
[11:43:13 INFO]: [CoreProtect] Enabling CoreProtect v21.2                                                             
[11:43:14 WARN]: java.lang.NumberFormatException: For input string: "0162000877589"                                                                                                                                                         
[11:43:14 WARN]:        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)                                                                                                                          
[11:43:14 WARN]:        at java.base/java.lang.Integer.parseInt(Integer.java:668)                                                                                                                                                           
[11:43:14 WARN]:        at java.base/java.lang.Integer.parseInt(Integer.java:784)                                                                                                                                                           
[11:43:14 WARN]:        at CoreProtect-21.2.jar//net.coreprotect.utility.Util.loadWorldEdit(Util.java:1022)                                                                                                                                 
[11:43:14 WARN]:        at CoreProtect-21.2.jar//net.coreprotect.config.ConfigHandler.performInitialization(ConfigHandler.java:432)
[11:43:14 WARN]:        at CoreProtect-21.2.jar//net.coreprotect.CoreProtect.onEnable(CoreProtect.java:75)                                                                                                                                  
[11:43:14 WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)                                                                                                                                                
[11:43:14 WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)                                                                                                                                  
[11:43:14 WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:536)                                                                                                                                 
[11:43:14 WARN]:        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:561)                                                                                                                                   
[11:43:14 WARN]:        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:475)
[11:43:14 WARN]:        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:633)                  
[11:43:14 WARN]:        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:419)                   
[11:43:14 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:306)                 
[11:43:14 WARN]:        at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1121)                          
[11:43:14 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302)
[11:43:14 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)          
[11:43:14 INFO]: [CoreProtect] CoreProtect has been successfully enabled!     
```

**Comment by vacla** [comment]
this should be fixed with #240

**Comment by amalon** [comment]
thanks. are there CI builds anywhere or would I have to build from source?
